sdks/java/container/build.gradle

```diff
@@ -82,3 +87,4 @@ docker {
 // Ensure that we build the required resources and copy and file dependencies from related projects
 dockerPrepare.dependsOn goBuild
 dockerPrepare.dependsOn copyDockerfileDependencies
+dockerPrepare.dependsOn copyDependencyLicenses
```

Comment:
Can this dependency be added to buildSrc/src/main/groovy/org/apache/beam/gradle/BeamModulePlugin.groovy in applyDockerNature?